Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -114,6 +119,22 @@ extension PaymentViewController {
         }
     }
 
+    private func setupCardNumberViewClosure() {
+        viewModel.updateCardNumberView = { [weak self] in
+            DispatchQueue.main.async {
+                guard let self = self else { return }
+                self.updateCardNumber()
+            }
+        }
+    }
+
+    private func updateCardNumber(){
+        guard let style = viewModel.paymentFormStyle?.cardNumber else { return }
+        cardNumberView.iconView.update(with: "icon-visa".image(forClass: CardListCell.self))
```

Comment:
I assume this is hardcoded for now as we are waiting for images from Alex designer.

Reply:
Hardcoded as we are waiting on scheme images and there is no validation happening yet